packages/veraswap-app/src/routes/index.tsx

```diff
@@ -364,14 +366,33 @@ function Index() {
     }, [remoteTransactionHash]);
 
     const importToken = (token: Token) => {
```

Comment:
Function currently gets instantiated on every render. Could be made cleaner by adding  `chainId` as a parameter and proper usage of `useCallback` (would need to add watchAsset, switchChain as deps)